runconfig/opts/parse.go

```diff
@@ -100,6 +100,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 		flStopSignal        = cmd.String([]string{"-stop-signal"}, signal.DefaultStopSignal, fmt.Sprintf("Signal to stop a container, %v by default", signal.DefaultStopSignal))
 		flIsolation         = cmd.String([]string{"-isolation"}, "", "Container isolation technology")
 		flShmSize           = cmd.String([]string{"-shm-size"}, "", "Size of /dev/shm, default value is 64MB")
+		flRuntime           = cmd.String([]string{"-runtime"}, "", "Runtime to use with the container")
```

Comment:
Should we support a short command `-r`? So it can be consistent with `containerd`